src/components/breadcrumb/Breadcrumb.tsx

```diff
@@ -9,7 +9,6 @@
  */
 
 import {ReactNode} from "react";
-import "styles/components/Breadcrumb.scss";
```

Comment:
Why is it removed, isnt it needed ?

Reply:
When we build the Dell Design React Common, the _webpack_ would generate all the styles in `main.css`. But whereas to export the component, we are building it from `tsc`, which in turn bundles the components and doesn't bundle the styles. So, we are not importing the style from components. If we are importing within the component, it would look for the `styles` folder in the build. Since we have all the styles in the `main.css`. So we have removed the style imports from the components.